docs/docs/reference/starlark-instructions.md

```diff
@@ -385,6 +402,26 @@ plan.assert(response["extract.second-element-from-list-head"], "==", "world")
 plan.wait(post_request_recipe, "extract.second-element-from-list-head", "==", "world")
 ```
 
+NOTE: `jq` returns a typed output that translates into the correspondent Starlark type. You can cast it using `jq` to match
+your desired output type:
+
+```python
+# Assuming response["body"] looks like {"url": "posts/1"}}
+post_request_recipe = PostHttpRequestRecipe(
+    ...
+    extract = {
+        "post-number": '.url | split ("/") | .[1]',
+        "post-number-as-int": '.url | split ("/") | .[1] | tonumber',
+    },
+)
+response = plan.request(
+    recipe = post_request_recipe,
+)
+# response["extract.post-number"] is "1" (starlark.String)
+# response["extract.post-number-as-int"] is 1 (starlark.Int)
+```
```

Comment:
Do we have a link somewhere to jq syntax already? If not, maybe it's a good place to add it. It's pretty advanced jq syntax already, if users stumble upon this and are curious to learn more about jq, having the link here might be handy. Deferring to you

Reply:
Done!